attendance_app/views.py

```diff
@@ -14,38 +15,65 @@ def __init__(self):
 		self.question = "What is 1 + 1"
 		self.answer = "2"		
 		self.views = "views/"
-	@csrf_exempt	
+	#@csrf_exempt	
 	def createForm(self,request):
-		html = loader.get_template(self.views + "create.html")
+		#html = loader.get_template(self.views + "create.html")
 		submitURL = request.scheme + "://" + request.get_host() + AppViews.path + '/submit' 
 		print(submitURL)
-		context = {"submitURL" : submitURL}
-		response = HttpResponse(html.render(context=context))
+		requestParam = request.GET or request.POST
+		instructor = UserProfile.objects.createUserProfile(requestParam.get('username'), requestParam.get('chat_url'),
+															requestParam.get('first_name'), requestParam.get('last_name'),
+															requestParam.get('email'), timezone.now(), requestParam.get('role'))
+		context = {"submitURL" : submitURL, "username" : requestParam.get('username'), "chat_url" : requestParam.get('chat_url')}
+		response = HttpResponse(render(request, self.views + "create.html", context))
 		print(response.getvalue())
 		return response	
 	# Create your views here.
-	@csrf_exempt
+	#@csrf_exempt
 	def submit(self,request): 
 		print (request.get_full_path())
 		submitResultURL = request.scheme  + "://" + request.get_host() + AppViews.path + '/submitResult'
-		context = {"question" : question, "submitResultURL" : submitResultURL}
-		html = loader.get_template(self.views + "question.html")		
-		response = HttpResponse(html.render(context=context))
-		return response	
+		requestParam = request.GET or request.POST
+		username = requestParam.get('username')
```

Comment:
Duplicated code between this section (line 37-43) and the below section (line 62 - 65)